CHANGES.md

```diff
@@ -15,7 +15,29 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
-[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+## Changes
+
+- THREDDS: add more options to configure catalog.xml
+  - Currently the default THREDDS configuration creates two default datasets, the Service Data dataset and the 
+    Main dataset. The Service Data dataset is used internally and hosts WPS outputs. The Main dataset is the
+    place where users can access data served by THREDDS. Both of these are configured to serve files with the following 
+    extensions: .nc .ncml .txt .md .rst .csv
+
+  - In order to allow the THREDDS server to serve files with additional extensions, this introduces two new
+    variables: 
+    - `THREDDS_SERVICE_DATA_EXTRA_FILE_FILTERS`: this allows users to specify additional [filter 
```

Comment:
This will most probably need to be 2 variables to align with Magpie definitions.